### ISSUE TITLE
 new Sound callback error is not executed when the sound path is incorrect.

### ISSUE THREAD

**Issue by MehdiJarraya** [zmxv/react-native-sound#507]
Hi guys,
I tested this issue with react-native-sound 0.10.9 and 0.10.4 but the problem still persist.
```javascript
 finalPist = new Sound(incorrectPath, null, (error) => {
            if (error) {
//This block is never executed 

            }
            else {
            }
          })
```

**Comment by afilp** [comment]
I have the problem of sounds not playing at all anymore, they used to be playing.

**Comment by shanzhi** [comment]
same as you

**Comment by paulmelnikow** [comment]
#511 includes a fix for this on Android.

Are you having the problem on iOS too?

**Comment by paulmelnikow** [comment]
It looks like this was fixed in #405, at least for Android.
